go/teams/chatmessaging.go

```diff
@@ -2,25 +2,18 @@ package teams
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/keybase/client/go/chat/globals"
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/protocol/chat1"
 	"github.com/keybase/client/go/protocol/keybase1"
 )
 
-func SendTeamChatWelcomeMessage(ctx context.Context, g *libkb.GlobalContext, team, user string) (res bool) {
-	var err error
-	defer func() {
-		if err != nil {
-			g.Log.CWarningf(ctx, "failed to send team welcome message: %s", err.Error())
-		}
-	}()
-
+func SendTeamChatWelcomeMessage(ctx context.Context, g *libkb.GlobalContext, team, user string) (err error) {
```

Comment:
Ah I see you did all this for the `Warning` above. I don't think it makes any sense to do it that way.